CONTRIBUTING.md

```diff
@@ -1,44 +1,116 @@
+- `Needs CSS/Design/Research` - Issues that require any work that would be completed outside of this repo related to css, design or research. For example some component contributions will require updates to css in the core [patternfly](https://github.com/patternfly/patternfly) repo.
+- `Assigned` - Issues that are ready to be worked and are already assigned.
+- `In Progress` - Issues that are actively being worked.
+- `PR in Review` - Issues for which a PR is open and are ready for review.
+
+Issues can be manually added to the project board from either the issue page or the project board page.
+
+## Contribution Process
+
+### Creating Issues for Bugs
 
-## React Component Contribution Process
+If you find a bug within the repo, please help us track the issue by following these steps:
 
-React component contributions should observe the following high level process:
+1. In the issue tracker, check if the bug has already been reported.
+  - If it does exist, but doesn’t capture key details that you have noticed, please add a comment to the existing bug.
+2. Create an issue if one doesn’t already exist.
+  - Provide details like what component is affected, steps to reproduce, and any other information that is critical to addressing the issue
+  - Assign the label “bug” to the issue
 
-1. Check the [PatternFly Decision Tree](https://github.com/patternfly/patternfly-design/blob/master/resources/decision-tree/PatternflyDecisionTree.pdf) to see if your component meets the criteria for an addition to PatternFly. If you're unsure, reach out to us on the mailing list!
-2. Create an `enhancement` issue to indicate you'd like to contribute this component to PatternFly React.
-  - At this time, the PatternFly React design team will review the enhancement issue and verify that the component passes the decision tree. Only components that pass will be accepted into the repo. If the component does not pass, the team will work with you on identifying any sub-components that can be extracted that would be needed by more than one project.
+### Creating Issues for New Components
+
+To create an issue for adding a new component to the repo, please observe the following process:
+
+1. If the component does not yet exist as a PatternFly component, check the [PatternFly Decision Tree](https://github.com/patternfly/patternfly-design/blob/master/resources/decision-tree/PatternflyDecisionTree.pdf) to see if your component meets the criteria for an addition to PatternFly. If you're unsure, reach out to us on the patternfly@redhat.com mailing list.
+2. If the component exists in PatternFly or passes the Decision Tree, then create an issue with the following details:
+  1. Assign the label `enhancement` to the issue
+  2. Include the text “Component -“ in the beginning of the title if the issue captures a new component
+  3. If the component currently exists in patternfly, include a link to it.
```

Comment:
Could we reword this to 

If the component is documented as a [PatternFly Pattern Library](http://www.patternfly.org/pattern-library/), include a link to it.

Reply:
👍